Bullwinkle.class.nut

```diff
@@ -414,7 +455,7 @@ class Bullwinkle {
     // message timeouts.
     function _watchdog() {
         // Get the current time
-        local t = time();
+        local t = Bullwinkle._isAgent() ? time() : hardware.micros()/1000000
```

Comment:
Specifically, this change and the one on line 479, are causing messages not to be timed/fail when hardware.micros goes negative...